app/models/user.rb

```diff
@@ -155,6 +157,23 @@ def assigned_to_postcode?(postcode)
 
   protected
 
+  def partner_in_neighbourhood_scope?(partner_id)
+    neighbourhood_admin? &&
+      (
+        owned_neighbourhood_ids & (
+          Partner.find_by(id: partner_id).owned_neighbourhood_ids
+        )
+      ).any?
+  end
+
+  def partner_in_partnership_scope?(partner_id)
```

Comment:
seems like a really good idea to split these, if at some point in the future we just have partnership admins who aren't neighborhood admins this will be less painful